.changeset/green-bags-hug.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': patch
+---
+
+Implement support for formatting Openshift dashboard url links
```

Comment:
OK so now Vale says (correctly!) that this should be OpenShift with a capital S :)

Reply:
Oups! Fixed.